Dockerfile

```diff
@@ -0,0 +1,13 @@
+# syntax=docker/dockerfile:1
+FROM --platform=linux/amd64 synthetixio/docker-e2e:18.16-ubuntu as base
+
+RUN mkdir /app
+WORKDIR /app
+
+RUN apt update && apt install -y nginx
```

Comment:
is there an upstream that has nginx installed? that would save some time in the job.

Reply:
don't think there is one by synthetixio. we're using their images at the moment to avoid any problems with missing or conflicting dependencies.
We can look into either using another image for this or creating a custom image for our own requirements